### DISCUSSION TITLE
openmptcprouter: mptcp filtered by mobile operator even if no warnings

### DISCUSSION THREAD

**Discussion by muko82** [Ysurac/openmptcprouter#3151] [General]
Hello,

after few tests I've realized that Vodafone in Italy is really filtering MPTCP (I also receive a warning on the webpage of openmptcprouter).
Enabling VPN solve the situation increasing performances.

In the test environment I'm using two SIM.
I noticed from the bandwidth graphs that the first one (another mobile operator) wan't used (zero traffic on it).
On the MTPTCP status page I had no warning about multipath filtering.

I enabled then the VPN on both SIM and magically... :-)
So the question is: is it possible that the provider is filtering multipath even I don't get warnings?

How it's possible that?

Thanks!

**Comment by Ysurac** [comment]
What is the version used ? With kernel 5.4 or snapshot with kernel 6.1 ?

**Reply by muko82** [reply]
The image download from the website: https://www.openmptcprouter.com/download
The one for Raspberry 64bit (4b) sysupgrade ext4, while for the VPS I used the script

**Reply by Ysurac** [reply]
Try snapshot image using kernel 6.1 on router and VPS part, this is sometimes less filtered.

**Reply by muko82** [reply]
Mmmm ok, but the snapshot with 6.1 is in test...

**Comment by Melania95** [comment]
> Hello,
> 
> after few tests I've realized that Vodafone in Italy is really filtering MPTCP (I also receive a warning on the webpage of openmptcprouter). Enabling VPN solve the situation increasing performances.
> 
> In the test environment I'm using two SIM. I noticed from the bandwidth graphs that the first one (another mobile operator) wan't used (zero traffic on it). On the MTPTCP status page I had no warning about multipath filtering.
> 
> I enabled then the VPN on both SIM and magically... :-) So the question is: is it possible that the provider is filtering multipath even I don't get warnings?
> 
> How it's possible that?
> 
> Thanks!

Hi, did you manage to solve the problem? Even with 2 Vodafone sims I don't get the speed I hoped for...probably for the same reason as you. By activating the vps for both the performances are really low... have you by chance installed the kernel version 6.1? Was it better? Thanks and greetings.
Martina

**Reply by muko82** [reply]
Yes, I did it by using the VPN on Vodafone.
After that I did some tune of the parameters reaching the speed of 110Mbit/s in download and 60Mbit/s in upload. 

Pretty good if considering that I was using raspberry 4 with two old android mobile phone.

**Reply by Melania95** [reply]
So do you recommend me to install kernel version 6.1 because it performs better?

**Reply by muko82** [reply]
My tests ended in the first week of March. I tested both kernel but with 6.1 I had a very bad experience: it was unstable and with poor performances.

Now I understood that there's a new release: maybe it's time to test again.

It took me a long time to find the perfect configuration...

**Reply by Melania95** [reply]
thanks for the answers...I basically only have a problem with version 0.59 with kernel 5.4 the OMR bypass! In practice what happens to me is that if I insert the link for example "google.it" when I enter that site to be bypassed I come out of the bonding! If I do a speedtest it doesn't give me the VPS anymore but I browse on one of the sims normally...Do you know why this? Non i have see in the new update there are some fix about OMRBYPASS...

**Reply by muko82** [reply]
Nope.

Sorry but one bypass isn't useful for my purposes: I need bonding to upload pictures and streaming at any time. I did tested it

**Reply by Melania95** [reply]
One last thing...if I install version 0.60 with kernel 5.4 do I also have to update the VPS or only by installing kernel 6.1 should I also update the VPS? Thanks for your patience...

**Reply by muko82** [reply]
You must install the VPS with proper script. You'll find it on the website.

Otherwise it will not work

**Reply by Melania95** [reply]
Yes, I already have everything installed and working (VPS+OMR 0.59.1 with kernel 5.4) but I haven't understood whether I should only install the updated version 0.60 again with kernel 5.4 and leave the VPS as I have it now or update that too ..
